webapp/src/components/rhs/rhs_new_tab.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {useDispatch} from 'react-redux';
+import styled from 'styled-components';
+
+import {
+    FormatListNumberedIcon,
+    LightbulbOutlineIcon,
+    PlaylistCheckIcon,
+} from '@mattermost/compass-icons/components';
+
+import {createPostImmediately} from 'mattermost-redux/actions/posts';
+
+import RHSImage from '../assets/rhs_image';
+
+import {Button} from './common';
+
+const CreatePost = styled((window as any).Components.CreatePost)`
+    padding: 0px;
+`;
+
+const OptionButton = styled(Button)`
+    color: rgb(var(--link-color-rgb));
+    background-color: rgba(var(--button-bg-rgb), 0.04);
+    svg {
+        fill: rgb(var(--link-color-rgb));
+    }
+`;
+
+const NewQuestion = styled.div`
+    padding: 12px;
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    .AdvancedTextEditor {
+        padding: 0px;
+    }
+`;
+
+const QuestionTitle = styled.div`
+    font-weight: 600;
+    font-size: 22px;
+`;
+
+const QuestionDescription = styled.div`
+    font-weight: 400;
+    font-size: 14px;
+`;
+
+const QuestionOptions = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+`;
+
+const PlusMinus = styled.i`
+    width: 14px;
+    font-size: 14px;
+    font-weight: 400;
+    margin-right: 4px;
+`;
+
+type Props = {
+    botChannelId: string
+    selectPost: (postId: string) => void
+    setCurrentTab: (tab: string) => void
+}
+
+const setEditorText = (text: string) => {
+    const replyBox = document.getElementById('reply_textbox');
```

Comment:
I guess this works, but maybe we should have an official way to do this?

Reply:
This was the less intrusive way that I found to do it. I think would be great to have an official way to do it, I don't know if this PR should solve that. Again this PR is already complex enough, we can polish things later, but I don't think we should polish everything here.